.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,6 +10,7 @@
   ./mvnw clean package 
   ./bin/generate-samples.sh
   ./bin/utils/export_docs_generators.sh
+  ./bin/utils/export_generators_readme.sh
```

Comment:
what about simply run `./bin/utils/ensure-up-to-date` instead?

if i remember correctly, `./bin/utils/ensure-up-to-date` was suggested before but it took a while to run...

Reply:
Ok, running it every single time will be wasteful for most. It actually only becomes interesting in case you create a new generator. Let me be more precise in the README